src/Components/v2/RecentlyViewed.tsx

```diff
@@ -67,6 +68,22 @@ export class RecentlyViewed extends React.Component<RecentlyViewedProps> {
                       />
                     )
                   }}
+                  renderLeftArrow={({ Arrow }) => {
+                    return (
+                      <ArrowContainer>
+                        <Arrow />
+                      </ArrowContainer>
+                    )
+                  }}
+                  renderRightArrow={({ Arrow }) => {
+                    return (
+                      <ArrowContainer>
+                        {me.recentlyViewedArtworks.edges.length > 4 && (
```

Comment:
@damassi Ah we should expose the `checkLastItemVisible` so we don't need to do this type of thing. The carousel should decide to show the arrow based on what's still visible. Could scope for another PR.

Reply:
Sounds good -- there are a few spots around Reaction that we could update all at once (in diff PR, as mentioned)